tests/test_get_value_of_pool_position.cairo

```diff
@@ -0,0 +1,515 @@
+    let two = Math64x61.fromFelt(2);
+    let half = Math64x61.div(one, two);
+
+    // Calls
+    let (_) = IAMM.trade_open(
+        contract_address=amm_addr,
+        option_type=0,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=0,
+        option_size=one,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    
+    let (_) = IAMM.trade_open(
+        contract_address=amm_addr,
+        option_type=0,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=1,
+        option_size=half,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+
+    );
+
+    // Puts
+    let (_) = IAMM.trade_open(
+        contract_address=amm_addr,
+        option_type=1,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=0,
+        option_size=one,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    
+    let (_) = IAMM.trade_open(
+        contract_address=amm_addr,
+        option_type=1,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=1,
+        option_size=half,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    let (pools_pos_val_call_6) = ILiquidityPool.get_value_of_pool_position(
+        contract_address = amm_addr,
+        lptoken_address = lpt_call_addr
+    ); 
+    // Pool is net short -> locked_capital - premia - fees
+    // 0.5 - 0.0016825104937091434 - 0.0016825104937091434 * 0.03
+    assert pools_pos_val_call_6 = 1148925511395203630;
+        
+    let (pools_pos_val_put_6) = ILiquidityPool.get_value_of_pool_position(
+        contract_address = amm_addr,
+        lptoken_address = lpt_put_addr
+    ); 
+    // Pool is net short -> locked_capital - premia - fees
+    // 0.5*1500 - 53.17407585832186 - 53.17407585832186 * 0.03
+    assert pools_pos_val_put_6 = 1603092853688281053408;
+
+    // Close all positions
+    let (_) = IAMM.trade_close(
+        contract_address=amm_addr,
+        option_type=0,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=0,
+        option_size=one,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    let (_) = IAMM.trade_close(
+        contract_address=amm_addr,
+        option_type=0,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=1,
+        option_size=half,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    let (_) = IAMM.trade_close(
+        contract_address=amm_addr,
+        option_type=1,
+        strike_price=strike_price,
+        maturity=expiry,
+        option_side=0,
+        option_size=one,
+        quote_token_address=myusd_addr,
+        base_token_address=myeth_addr
+    );
+    let (_) = IAMM.trade_close(
```

Comment:
could you close half of the options? half of each one check the get_value_of_pool_position and then the rest to continue the test

Reply:
moment

Reply:
done